core/system/inc/secure_gateway.h

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
@AlessandroA do we still need the Secure gateway definition? Can't we use that into a dead branch till further notice ?